CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ set(SOURCE_FILES
         src/interpreter/vm/VM.cpp)
 
 set(TEST_FILES
-        test/TestLexer.cpp test/TestParser.cpp)
+        test/TestLexer.cpp test/TestParser.cpp src/include/DurianFunction.h)
```

Comment:
`.h` files don't need to be included in the CMakeLists (the new version of CLion fixes that whole "this file is not part of the project" thing 😛)